Solr docker configuration

sdreher opened the pull request:
* Add the basic footprints solr core configuration to the repo
* Provide some basic docker commands to get solr up & running locally.

Instructions added to the repo wiki: https://github.com/ccnmtl/footprints/wiki/Local-Solr-Instance

nbuonin commented:
@sdreher Looks good to me! Some ideas:
* Would you ever need to access the shell in the Solr container? It might be handy to have a target for that.
* If this would ever run in testing, it might add `-` before each command in the clean target so it doesn't error if there's no container to stop/remove
* I don't think there would ever be files that match these target names, but it might be good to add them to a `.PHONY: ...`

sdreher commented:
Thanks for the close review! I have:
* added a shell target.
* added the `-`, and also updated the target name to `docker-solr-clean`, which matches the terminology in the drupal-docker.mk you setup.
* added the targets to `.PHONY`
